### ISSUE TITLE
Cannot use s3 upload action from a public screen - session not authenticated

### ISSUE THREAD

**Issue by mikesealey** [Budibase/budibase#14952]
**Checklist**
- [x] I have searched budibase discussions and github issues to check if my issue already exists

**Hosting**
- Cloud
  - miketest1

**Describe the bug**
In a public screen I have a form that includes an "Upload File to S3" action. When performing this action any logged-in app user can do it fine, but from a public screen as an unauthenticated user this is not possible and throws an error.

**To Reproduce**
1. Set up your S3 data connection
2. In the design tab, add a form component
3. add a nested s3 file upload component
4. add a button with an on-click action to upload file to S3

**Expected behavior**
Should be able to perform this action as a public user.

**Screenshots**
https://jam.dev/c/c194594c-fb99-4f6b-b710-01e0436c4973
  
**App Export**
Export available on request

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser Chrome
 - Version 130.0.6723.92 (Official Build) (arm64)

**Additional context**
Bug discovered by customer, reported on their behalf by CSE

**Comment by deanhannigan** [comment]
Hey @mike.sealey, S3 upload requires an authenticated budibase session in order to generate the upload URLs. We don't currently support public upload via S3

**Comment by mikesealey** [comment]
I was working on this case with @andrew.thompson who suggested that it should be possible. The example given was submitting a photo for a driving license renewal. Users are not logged in, but can submit that file.<br><br>If this is something that is deliberately unavailable perhaps we could handle this better - maybe something as simple as a note in the builder.

**Comment by deanhannigan** [comment]
Agreed, we should highlight this somewhere in the builder. The pre-signed URL generation required for the S3 file upload field requires an authenticated session.
